### ISSUE TITLE
New Term Request: Engineering design, Scientific Instrument Design

### ISSUE THREAD

**Issue by only1chunts** [FAIRsharing/subject-ontology#64]
This is more a request to see if a new term would be appropriate for this.

### Label
**Recommendation**:  Hardware design ?

**Reasoning**: 
A specific example for the use case would be:
https://doi.org/10.1093/gigascience/giab044

To add subject classifications to this manuscript there are all the obvious ones for its biological content, but the part I'm struggling with is the addition of a subject to cover the design of a physical monitoring device that they made for the study and have provided plans for the replication of it by others. We've initially called it "Hardware" but thats just because we dont have any idea of a more appropriate label!

Another example might be where the authors have designed and 3D printed a piece of hardware to carry out a specific task in their research and are sharing the specifications in the manuscript.

Are there any existing subjects that cover the design and creation of bespoke devices?

**Comment by allysonlister** [comment]
Thanks for an interesting question! While we have _Materials Engineering_, that might be a bit too high level for what you need. Some of our sources include a term _Materials Design_ as a child of _Materials Engineering_, and this could be the right level for what you need. 

Out of curiosity, are you making use of SRAO, or perhaps taking a look at the hierarchy as a possible solution to your issue? We always love to hear how we might be of help.

Thanks very much!
Allyson

**Comment by only1chunts** [comment]
materials design might be suitable depending on its definition? but it sounds like its probably more to do with the design of the material rather than the design of an instrument/object made from that material.
I cant see it in the OLS at EBI yet so I assume its not actually included in SRAO yet?
We are moving towards using an SRAO slim for journal article and associated dataset classification in GigaByte and GigaDB respectively. I can send you the current slim we're trying to incorporate if its of interest, just drop me an email to chris (at) gigasciencejournal.com

**Comment by allysonlister** [comment]
That's fantastic, Chris!

It's not in SRAO yet - we'll add it once we're sure what we're adding :-) We can only do that once we're sure what we're describing. Below is a short survey I've done with a variety of vocabularies. You can use these as a starting point for describing exactly what you'd like to be included in SRAO - I've included the template for the definition, name and hierarchy of your term suggestion below.

* PLOS Thesaurus has _Materials Design_ as a child of _Materials Science_, but no definition, so we don't know if it's a good fit for you. 
* NCIT has a definition for [_Design_](https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C43358) ("The act of working out the form of something (as by making a sketch or outline or plan) or the result of this activity") but its hierarchy is unhelpful for our purposes. 
* SIO has [_Design_](https://www.ebi.ac.uk/ols/ontologies/sio/terms?iri=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000705) as a child of ICE and, more narrowly, specification. That means it's using design in the "design document" sense rather than the research area that covers design, if you get my meaning. So, if I'm understanding your use case (and how we would implement it in SRAO), then we can't use that as a starting point either. 
* [ERO](https://www.ebi.ac.uk/ols/ontologies/ero/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FERO_0001294&viewMode=All&siblings=false) models designs as _planned process_ -> _technique_. 
* bepress has a hierarchy but no persistent identifiers or definitions. It's relevant design term is only really _Engineering: Mechanical Engineering: Computer-Aided Engineering and Design_, however this is only for CAD.
* [AgroVoc](https://agrovoc.fao.org/browse/agrovoc/en/page/c_2208) points to [USDA Thesaurus](https://agclass.nal.usda.gov/mtwdk.exe?k=default&l=60&s=5&t=2&w=12660), whose definition might be closer to what you need, but I think the hierarchies aren't quite there.

To be honest, I haven't found a good fit among my usual suspects of various vocabularies, so if you are similarly unsuccessful we may need to mint our own IRIs.

I suggest you consider carefully what definition you would like, and where you think it should belong in the hierarchy, remembering that this is a hierarchy of research areas and therefore the definitions should match. If you want _Hardware Design_ then you'll need to provide a definition of it that describes what you mean by hardware please as well as where you think it should belong in the hierarchy. If you would like to include it in a _Materials Design_ hierarchy, then a definition and placement of that term within the existing SRAO hierarchy should also be supplied.

Please would you provide the information below for each term you might like to include within SRAO, and we can make a decision about inclusion.

Thanks very much for contacting us!

## Term Name 1


### Definition
Either state that you are happy with the definition provided by the reference ontology, or provide your own definition here, with appropriate attribution (e.g. created entirely by you, modified by you from a 3rd party such as the OED or Wikipedia).

### Hierarchy
**Recommendation**: list where in the hierarchy your term should go / which terms its parent(s) should be.

**Reasoning**: Why this hierarchy is your preferred choice.

**Comment by only1chunts** [comment]
After looking at the various places you suggested above I'm thinking maybe the PLOS thesaurus term "[Open source hardware](https://bioportal.bioontology.org/ontologies/PLOSTHES/?p=classes&conceptid=http%3A%2F%2Flocalhost%2Fplosthes.2017-1%2311878)" might be a good term for us.
I would suggest it fits under the main grouping of "Engineering Science", and possibly within the "Production Technology" node

Do you think that the term "Open source hardware" fits within the SRAO model? If so, do you agree with my suggestion of its position within the hierarchy?

**Comment by allysonlister** [comment]
As long as you are happy describing the hardware itself (and not the design of it), then a term such as _Open source hardware_ would be suitable for your needs.

However, such a term is not a research area, rather it is a "thing", what in FAIRsharing we would normally put within our [domain ontology](https://github.com/FAIRsharing/domain-ontology). However, the production and curation of our domain ontology is more automated than with SRAO, and we tend to only add terms that FAIRsharing users have directly requested. This results in a completely fit-for-purpose terminology for FAIRsharing, but one which may not be as suitable for our community (such as you). 

In short, we're very happy for you to use DRAO, but you may not find it as suited to your needs as SRAO. Alternatively, if you can request a term that describes a research area, we should be able to add it to SRAO.

Please let me know if I am being clear - happy to explain or discuss more, either with a call or via this thread.

Thanks very much!

**Comment by only1chunts** [comment]
I'm not sure how you differentiate between a subject and domain, but if "open source hardware" is not a subject maybe "Hardware design" is a better choice? Or perhaps "scientific instrument design"?
From Wikipedia the definition of **Design** is - 

> A design is a plan or specification for the construction of an object or system or for the implementation of an activity or process, or the result of that plan or specification in the form of a prototype, product or process.

**Scientific instrument** is - 

> A scientific instrument is a device or tool used for scientific purposes, including the study of both natural phenomena and theoretical research.

Put them together to get
## Scientific Instrument Design
### Definition
The planning or specification for the construction of an object, system, device or tool used for scientific purposes, including the study of both natural phenomena and theoretical research. May include the result of the plan or specification in the form of a prototype, product or process.
### Hierarchy
Child of Engineering Science

**Comment by allysonlister** [comment]
I understand that where we draw the line between subject and domain might not be clear at first. In short, we are aiming to have high-level research areas in SRAO, and everything else in DRAO. 

I think this is a great addition to SRAO. We might want to introduce an intermediate class called, e.g., Engineering Design. However, we are planning to freeze FAIRsharing and start our data migration to the new version of the site next week. What sort of timescale do you need for the incorporation of this term?

**Comment by only1chunts** [comment]
we are not actually planning on linking data using these terms for a long time yet so the fact it's not "in" SRAO yet isn't a problem if the term name is now in the pipeline and likely to be included at some point in 2021, we can just add it to our slim and worry about linking to SRAO later.
The intermediary class Engineering Design sounds like a good idea.

**Comment by allysonlister** [comment]
Sounds good. I'll link it to our next milestone, and aim to put it in with the next update. Probably late Autumn right now.

Thanks!

**Comment by allysonlister** [comment]
Looking more closely over the hierarchy to add this term, I think that rather than creating Engineering Design, we'll place this term as follows:

Engineering Science -> Mechanical Engineering (https://www.ebi.ac.uk/ols/ontologies/srao/terms?iri=http%3A%2F%2Faims.fao.org%2Faos%2Fagrovoc%2Fc_4682) -> Scientific Instrument Design.

The definition of Mechanical Engineering suits this placement
> Mechanical Engineering applies the principles of engineering, physics and materials science to the design, analysis, manufacture and maintenance of mechanical systems. 

I'll go ahead with this, but please let me know if you have any reservations and we can modify.

**Comment by only1chunts** [comment]
that sounds like a good plan, thanks.
